prometheus/src/main/java/org/opensearch/sql/prometheus/functions/scan/QueryExemplarsFunctionTableScanOperator.java

```diff
@@ -0,0 +1,75 @@
+package org.opensearch.sql.prometheus.functions.scan;
+
+import java.io.IOException;
+import java.security.AccessController;
+import java.security.PrivilegedAction;
+import java.util.Locale;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.json.JSONArray;
+import org.opensearch.sql.data.model.ExprValue;
+import org.opensearch.sql.executor.ExecutionEngine;
+import org.opensearch.sql.prometheus.client.PrometheusClient;
+import org.opensearch.sql.prometheus.functions.response.QueryExemplarsFunctionResponseHandle;
+import org.opensearch.sql.prometheus.request.PrometheusQueryExemplarsRequest;
+import org.opensearch.sql.storage.TableScanOperator;
+
+@RequiredArgsConstructor
+public class QueryExemplarsFunctionTableScanOperator extends TableScanOperator {
```

Comment:
Can you merge this one with `QueryRangeFunctionTableScanOperator`? Prometheus Table Scan Operator can contain a query reference as it is done for OpenSearch.

Reply:
And/Or add a javadoc